server/tests/controllers/category.test.js

```diff
@@ -494,5 +472,22 @@ describe('Categorizes articles', () => {
           done();
         });
     });
+
+    // Users should be able to get the articles for a category
+    it('Returns the articles in a category', (done) => {
+      chai.request(app)
+        .get('/api/article-categories/Technology/articles')
+        .set('Accept', 'application/json')
+        .set('Authorization', `Bearer ${userToken}`)
+        .set('Content-Type', 'application/json')
+        .end((req, res) => {
+          expect(res).to.have.status(200);
+          expect(res.body.status).to.equal('success');
+          expect(res.body).to.have.property('category');
+          expect(res.body.category).to.have.property('createdAt');
```

Comment:
It is always better to use object destructuring(lines 485 - 488). Also applies for lines 72 - 74 of your readingstats test file, lines 89 & 90.

Reply:
I'll do that now.